components/FileSelector.vue

```diff
@@ -1,37 +1,69 @@
 <template>
-  <v-file-input
-    v-model="files"
-    :multiple="multiple"
-    :label="label"
-    :accept="accept"
-    :rules="[(value) => !!value || 'The file is mandatory']"
-    color="primary"
-    chips
-    counter
-    show-size
-    @click:clear="stepper_tree.files = []"
-  />
+  <v-row>
+    <v-col class="pa-0">
+      <v-file-input
+        v-model="files"
+        :multiple="multiple"
+        :label="label"
```

Comment:
Is label really a FileSelector props or should it be hard coded?

Reply:
It changes depending if we have multiple true or false, maybe it should depend on that

Reply:
yes!